src/Command/SynchronizeVariantConfigurationCommand.php

```diff
@@ -74,10 +80,29 @@ protected function initialize(InputInterface $input, OutputInterface $output): v
 
     protected function execute(InputInterface $input, OutputInterface $output): int
     {
-        $this->variantConfigurationSynchronizer->synchronize();
+        $skipSetup = true !== $input->getOption(self::OPTION_SKIP_SETUP);
+
+        $synchronizeResult = $this->variantConfigurationSynchronizer->synchronize($skipSetup);
 
-        $this->io->success('Variant configuration synchronized');
+        self::reportSynchronizationResult($synchronizeResult, $this->io);
 
         return 0;
     }
+
+    public static function reportSynchronizationResult(VariantConfigurationSynchronizationResultInterface $synchronizeResult, SymfonyStyle $io): void
+    {
+        if (!empty($synchronizeResult->getMessages())) {
+            $io->writeln('Messages from synchronization');
+            $io->definitionList(...$synchronizeResult->getMessages());
+        }
+
+        foreach ($synchronizeResult->getSetupResults() as $setupResult) {
+            if (empty($setupResult->getMessages())) {
```

Comment:
Maybe the SetupResult should have a `hasMessages` method?

Reply:
Good idea